src/machine/actions.ts

```diff
@@ -0,0 +1,41 @@
+import { assign } from "xstate"
+import assignedForm from "../components/ClaimReview/form/assignedForm";
+import reportedForm from "../components/ClaimReview/form/reportedForm";
+import { reviewTaskMachineContext } from "./context";
+import { AssignEvent, PublishEvent, ReportEvent } from "./events";
+
+export const assignedUser = assign<reviewTaskMachineContext, AssignEvent>((context, event) => {
+    
+
+    return {
```

Comment:
Apparently we can make a single function and pass `formUi` as a parameter

Reply:
it's doesn't works because we need event typing, like: <ReviewTaskMachineContext, AssignEvent>